apps/meteor/app/apps/server/converters/threads.ts

```diff
@@ -0,0 +1,203 @@
+import { isEditedMessage, type IMessage } from '@rocket.chat/core-typings';
+import { Messages } from '@rocket.chat/models';
+import type { IMessage as AppsEngineMessage } from '@rocket.chat/apps-engine/definition/messages';
+import type { IRoom } from '@rocket.chat/apps-engine/definition/rooms';
+
+import { transformMappedData } from '../../../../ee/lib/misc/transformMappedData';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+interface Orchestrator {
+	rooms: () => {
+		convertById(id: string): Promise<unknown>;
```

Comment:
Curious as to why you preferred "casting" the return type when using the method instead of typing the return here :thinking: